test/itex/test_tensorflow_itex_basic.py

```diff
@@ -258,8 +258,11 @@ def test_itex_benchmark_gpu(self):
             evaluator('performance')
 
         found_multi_instance_log = False
+        num_of_instance = int(os.environ.get('NUM_OF_INSTANCE'))
```

Comment:
we should remove all environment variable usage from our code.

is this the only way to get num_of_instance and core_per_instance?